server/controllers/events/events_controller_e2e_test.go

```diff
@@ -666,7 +667,10 @@ func setupE2E(t *testing.T, repoDir string) (events_controllers.VCSEventsControl
 	e2eStatusUpdater := &events.DefaultCommitStatusUpdater{Client: e2eVCSClient}
 	e2eGithubGetter := mocks.NewMockGithubPullGetter()
 	e2eGitlabGetter := mocks.NewMockGitlabMergeRequestGetter()
-	tempchan := make(chan *models.TerraformOutputLine)
+	tempchan := make(chan *models.ProjectCmdOutputLine)
+	projectCmdOutputHandler := &handlers.DefaultProjectCommandOutputHandler{
```

Comment:
For tests you should generate mocks and use them instead of directly creating `handlers.DefaultProjectCommandOutputHandler` object.
Same way we are mocking WebsocketHandler